Time based recovery fixes (#1068)
BitcoinDesign/Guide: guide/savings-wallet/time-based-recovery.md

#### Wallet backup 

> After the wallet has been created, our application prompts the user to download the wallet backup file. This is important because, for multi-key wallets, it is not enough to back up the individual private keys to recover the wallet. You need at least all the extended public keys as well.

Reviewer: ```suggestion
After the wallet has been created, our application prompts the user to download the wallet backup file. This is important because, for multi-key wallets, it is not enough for the user to back up their own private key to recover the wallet. They need at least all the extended public keys as well.
```
Clarifying this sentence to make it more specific to the users individual situation. Otherwise you could argue that backing up XPUBs is not important, because they can be derived from the private keys again. Once we have more content about what's in a multi-key wallet backup, we can also link to that from here.

Author: That's a very good point. I hadn't thought of that option, to be honest. 😬🙏
